apps/system/sekey_test/Kconfig

```diff
@@ -0,0 +1,27 @@
+#
+# For a description of the syntax of this configuration file,
+# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
+#
+
+config SYSTEM_SEKEY_TEST
+	bool "Secure Key Tester"
+	default n
+	depends on SECURITY_LINK
```

Comment:
I think you need to add a dependency of mbedtls, CONFIG_NET_SECURITY_TLS.
If not, it brings build error because  this example uses mbedtls APIs.
I think `select CONFIG_NET_SECURITY_TLS` is better because user can enable this config only if user want to run this examples.
Please check this and add `depends on ` or `select` as your opinion here.

Reply:
@jeongarmy 
Ok. I will apply 'select' with mbedtls